apsi-enrollment-be/src/main/java/edu/pw/apsienrollment/attendance/AttendanceServiceImpl.java

```diff
@@ -50,17 +57,16 @@ public Attendance putIntoAttendanceList(Meeting meeting, User user) {
     public void markAsPresent(Long id, String token, Long meetingId) {
         Attendance attendance = attendanceRepository.findByIdAndTokenAndMeeting_Id(id, token, meetingId)
                 .orElseThrow(() -> new AttendanceNotFoundException(
-                        "Attendance not found",
-                        ImmutableMap.of("id", id, "token", token, "meetingId", meetingId)));
-        checkIfUserIsOrganizerOrSpeaker(attendance);
+                        "Attendance not found", ImmutableMap.of("id", id, "token", token)));
```

Comment:
Why is meetingId deleted?

Reply:
![632](https://user-images.githubusercontent.com/38384191/85235839-9a599b80-b418-11ea-9d46-94b69dbc4904.jpg)